c-sdk/config/cygwin-wrapper

```diff
@@ -56,24 +59,24 @@ do
                         driveletter=${no_i%%:*}
                         i=-I${mountpoint}/${driveletter}/${pathname}
                     elif test "${pathname:0:1}" = "/" -a -e "${pathname}" ; then
-			i=-I`cygpath -m "${pathname}"`
+            i=-I$(cygpath -m "${pathname}")
                     fi
                 else
-                    eval 'leader=${i%%'${mountpoint}'/[a-zA-Z]/*}'
+                    leader=$(eval '${i%%'"${mountpoint}"'/[a-zA-Z]/*}')
                     if ! test "${leader}" = "${i}"; then
-                        eval 'pathname=${i#'${leader}${mountpoint}'/[a-zA-Z]/}'
-                        eval 'no_mountpoint=${i#'${leader}${mountpoint}'/}'
```

Comment:
Most of the changes are straightforward, but I am not certain about this one. To me it feels that the assignment to the value of the `eval` command is more natural but would be interested to hear your opinions.

Reply:
Also sorry about the whitespace pollution again. Eclipse really hates those tabs!

Reply:
Eclipse replaces the tab with spaces and this is a good practice because spaces are better visualised across different editors.

Reply:
> To me it feels that the assignment to the value of the `eval` command

In `bash` I have only used `eval` with applications generating shell commands but for a single assignment it is not clear to me why it is used. Is it working differently in `sh` compared to `bash`? If not, then you can try to remove the `eval`.